tests/core/pyspec/eth2spec/test/altair/rewards/test_inactivity_scores.py

```diff
@@ -112,7 +112,7 @@ def test_random_high_inactivity_scores_leaking(spec, state):
 
 @with_altair_and_later
 @spec_state_test
-@leaking(epochs=5)
-def test_random_high_inactivity_scores_leaking_5_epochs(spec, state):
```

Comment:
just noting why this changed: want the `previous_epoch` to be past `MIN_EPOCHS_TO_INACTIVITY_PENALTY` so that we see the effect of the leak on scoring (which suggests this test was buggy before -- see other comment about the suggestion I made for this decorator)

Reply:
right, also want it to be beyond the standard minimum leak thus 8